server/src/routes/index.ts

```diff
@@ -10,30 +10,382 @@ import passport from 'passport';
 import cors from 'cors';
 import cookieParser from 'cookie-parser';
 import { frontendOrigin } from '../configs';
+import { definition } from './swagger';
+
+import swaggerUI from 'swagger-ui-express';
+import swaggerJsdoc from 'swagger-jsdoc';
+
+const options: object = {
```

Comment:
Perhaps a more specific type here, too

Reply:
`swaggerJSDoc.OAS3Options` I think.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-jsdoc/index.d.ts

Reply:
This as well, thanks.